client/header/activity-panel/wordpress-notices.js

```diff
@@ -176,6 +179,12 @@ class WordPressNotices extends Component {
 		notices.classList.remove( 'woocommerce-layout__notice-list-hide' );
 		screenMeta && screenMeta.classList.add( 'is-hidden-by-notices' );
 		screenLinks && screenLinks.classList.add( 'is-hidden-by-notices' );
+
+		const noticePosition = notices.getBoundingClientRect();
+		if ( noticePosition.y < HEADER_HEIGHT ) {
+			window.scrollBy( 0, noticePosition.y - HEADER_HEIGHT );
```

Comment:
Would it suffice to always scroll to the top using `window.scrollBy( 0, -window.scrollY );`

Reply:
We should probably change this - the `x` and `y` properties aren't supported in IE or Safari: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

Reply:
@jeffstieler Thanks for catching that. I've updated the PR as suggested.